### DISCUSSION TITLE
[SFC] Same Game Tsume Game

### DISCUSSION THREAD

**Discussion by PsyK0p4T** [sanni/cartreader#562] [Need help]
I was playing around with "Same Game Tsume Game" carts :
![tsume](https://user-images.githubusercontent.com/87064902/195715252-5e61d43c-1e17-4cd8-b548-a43d7c609cc6.jpg)

Dumping the main cart was not a problem, I got the expected CRC (133e1c5b) for it.
Then I wanted to access the top module. Not sure if this one was supported yet or if it's the right way to do, but I tried to use "Satellaview BS-X" and then "Read Memory Pack".
The file generated under \SNES\ROM\MEMPACK\xxxx\MEMPACK.bs was actually the main cart rom again (CRC 133e1c5b).
Any hints ?

**Comment by sanni** [comment]
As far as I know the current code only works with the BS-X cartridge as adapter.

![BS-XCart_413ca0cd-3efc-4fe6-a496-432a0a1199ac-3687904321](https://user-images.githubusercontent.com/19169494/195787378-b9712720-1c62-4bcd-a485-caf5d6186edd.jpg)

**Comment by PsyK0p4T** [comment]
Got it.
I'll try to get one, and hope that the top cart connectors fit in the BS-X. If someone already tested this, let me know ;)

**Comment by PsyK0p4T** [comment]
Got some fresh news !

Finally found a BS-X, used it with this Tsume-game module, and it worked.
Well, I got a 1MB raw dump, so had to trim the rom to remove repeated data. But the dump process worked fine. 

So, the current code is able to dump both flash modules ("BSMC-AF" pcb) and rom ones ("BSMC-CR" pcb). Maybe just an improvement on the chip size detection would be cool, but it works !
